src/arrayMethodJoin.js

```diff
@@ -5,8 +5,22 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    const newSeparator = separator === undefined ? ',' : separator;
```

Comment:
I understand the temptation to provide one line solution but it is better to keep it multi-lined with ternary))
And what's more, you can use shortcut for checking whether the separator is defined

```suggestion
    const newSeparator = separator 
      ? ',' 
      : separator;
```